Package:     squidguard
Subject:     make it useable with Berkeley DB version 5.x
Author:      Joachim Wiedorn <ad_debian at joonet.de>
Bug-Debian:  http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=621376
Forwarded:   yes
Last-Update: 2011-05-31

Since the end of 2010 the is a new major version of Berkeley DB.
For use with squidguard it need this patch.

---

diff -urN s07/src/sgDb.c s08/src/sgDb.c
--- s07/src/sgDb.c	2008-07-14 20:29:41.000000000 +0200
+++ s08/src/sgDb.c	2011-05-31 20:06:26.328740739 +0200
@@ -114,7 +114,7 @@
     }
   }
 #endif
-#if DB_VERSION_MAJOR == 4
+#if DB_VERSION_MAJOR == 4 || DB_VERSION_MAJOR == 5
   if(globalUpdate || createdb || (dbfile != NULL && stat(dbfile,&st))){
     flag = DB_CREATE;
     if(createdb)
